pkgs/top-level/all-packages.nix

```diff
@@ -27172,9 +27172,9 @@ with pkgs;
   rsclock = callPackage ../applications/misc/rsclock { };
 
   rstudio = libsForQt5.callPackage ../applications/editors/rstudio {
-    boost = boost166;
-    llvmPackages = llvmPackages_7;
-    jdk = jdk8; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
+    boost = boost169;
```

Comment:
Is this override necessary now? 1.69 is the current default Boost in nixos-unstable.